src/constants.ts

```diff
@@ -83,53 +83,29 @@ export const ORG_PERMISSIONS = {
 export const GCM_SENDER_ID = Config.GCM_SENDER_ID;
 
 //parameter names for the data we send to Analytics
-export const ANALYTICS_MCID = 'cru.mcid';
-export const ANALYTICS_SCREEN_NAME = 'cru.screenname';
-export const ANALYTICS_SITE_SECTION = 'cru.sitesection';
-export const ANALYTICS_SITE_SUBSECTION = 'cru.sitesubsection';
-export const ANALYTICS_SITE_SUBSECTION_3 = 'cru.subsectionlevel3';
-export const ANALYTICS_PREVIOUS_SCREEN_NAME = 'cru.previousscreenname';
-export const ANALYTICS_APP_NAME = 'cru.appname';
-export const ANALYTICS_LOGGED_IN_STATUS = 'cru.loggedinstatus';
-export const ANALYTICS_SSO_GUID = 'cru.ssoguid';
-export const ANALYTICS_GR_MASTER_PERSON_ID = 'cru.grmasterpersonid';
-export const ANALYTICS_FACEBOOK_ID = 'cru.facebookid';
-export const ANALYTICS_CONTENT_LANGUAGE = 'cru.contentlanguage';
-export const ANALYTICS_SECTION_TYPE = 'cru.section-type';
-export const ANALYTICS_ASSIGNMENT_TYPE = 'cru.assignment-type';
-export const ANALYTICS_EDIT_MODE = 'cru.edit-mode';
-export const ANALYTICS_PERMISSION_TYPE = 'cru.permission-type';
+export const ANALYTICS_SCREEN_NAME = 'cru_screenname';
+export const ANALYTICS_SITE_SECTION = 'cru_sitesection';
+export const ANALYTICS_SITE_SUBSECTION = 'cru_sitesubsection';
+export const ANALYTICS_SITE_SUBSECTION_3 = 'cru_subsectionlevel3';
+export const ANALYTICS_PREVIOUS_SCREEN_NAME = 'cru_previousscreenname';
+export const ANALYTICS_APP_NAME = 'cru_appname';
+export const ANALYTICS_LOGGED_IN_STATUS = 'cru_loggedinstatus';
+export const ANALYTICS_SSO_GUID = 'cru_ssoguid';
+export const ANALYTICS_GR_MASTER_PERSON_ID = 'cru_grmasterpersonid';
+export const ANALYTICS_FACEBOOK_ID = 'cru_facebookid';
+export const ANALYTICS_CONTENT_LANGUAGE = 'cru_contentlanguage';
+export const ANALYTICS_SECTION_TYPE = 'cru_section-type';
+export const ANALYTICS_ASSIGNMENT_TYPE = 'cru_assignment_type';
+export const ANALYTICS_EDIT_MODE = 'cru_edit_mode';
+export const ANALYTICS_PERMISSION_TYPE = 'cru_permission_type';
 
-export const LOGGED_IN = 'logged in';
-export const NOT_LOGGED_IN = 'not logged in';
+export const LOGGED_IN = 'logged_in';
+export const NOT_LOGGED_IN = 'not_logged_in';
 
 export const ACTIONS = {
   PERSON_ADDED: {
-    name: 'Person Added',
-    key: 'cru.personadded',
-  },
-  STATUS_CHANGED: {
-    name: 'Contact Status Changed',
-    key: 'cru.contactstatuschanged',
-  },
-  EMAIL_ENGAGED: {
-    name: 'Contact Engaged by Email',
-    key: 'cru.emailiconengaged',
-  },
-  CALL_ENGAGED: {
-    name: 'Contact Engaged by Phone',
-    key: 'cru.calliconengaged',
-  },
-  TEXT_ENGAGED: {
-    name: 'Contact Engaged by Text',
-    key: 'cru.texticonengaged',
-  },
-  FILTER_ENGAGED: {
-    name: 'Search Filter Engaged',
-    key: 'cru.searchfilterengaged',
-  },
-  SEARCH_FILTER: {
-    key: 'cru.searchfilter',
+    name: 'person_added',
+    key: 'cru_personadded',
```

Comment:
A big thing I'm debating with is whether it would be worth it to remove these ```key``` values, as they are currently being passed as unnecessary parameters on action events. Sending the additional parameters does not affect anything, as they just ignore them. So it's just trying to decide if cleaning it up is worth the effort or not.

Reply:
Hmm... Simpler sounds better but we can take another pass at it later...